plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -253,6 +253,7 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			// used for pod eviction
 			rbacv1helpers.NewRule("update").Groups(legacyGroup).Resources("pods/status").RuleOrDie(),
 			rbacv1helpers.NewRule("list", "delete").Groups(legacyGroup).Resources("pods").RuleOrDie(),
+			rbacv1helpers.NewRule("get", "list", "create", "watch", "delete").Groups(networkingGroup).Resources("clustercidrconfigs").RuleOrDie(),
```

Comment:
Why is node-controller going to create or delete CCC ?

Reply:
No it won't, removed create, delete and watch permissions which are not required